index.html

```diff
@@ -263,7 +270,67 @@ <h3 class="about-title">Skills</h3>
         </div>
       </div>
     </div>
-      </section>
-    </main>
+    <div class="contact-form">
+      <nav class="text-layout">
+          <h4 class="form-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h4>
+      </nav>
+      <form method="post" action="https://formspree.io/f/mbjbdrnb" class="form-data">
+        <fieldset>
+          <label><input class="label" type="text" name="first-name" placeholder="Full name" maxlength="30" required /></label>
+          <label><input class="label" type="email" name="email" placeholder="Email address" required /></label>
+          <label>
```

Comment:
### Figma Design
![padding-top-left](https://user-images.githubusercontent.com/93039359/192833383-fdeaabc4-ebed-48f6-a1ed-a96eda0ad075.png)

- [ ] Kindly add some **padding top** and **padding left** in your input fields as shown in the Figma image above with red lines :+1:
- [ ] Your contact form has a horizontal scroll on 375px screen size which is an iphone screen size. Kindly make your design fit on all the mobile screens :+1:

Reply:
I'm just following the template.